uber/tests/import_test_data.py

```diff
@@ -18,7 +18,7 @@ def random_group_name():
 def import_groups(session):
     for g in dump['groups']:
         secret_id = g.pop('secret_id')
```

Comment:
Actually this should NOT change.  This is parsing a dump of last year's data, and I believe that ``amount_owed`` is actually the key in that ``test_data.json`` file or whatever it's called.